assignment1.dart

```diff
@@ -0,0 +1,124 @@
+import 'dart:io';
+
+enum type{
+  raw , manufactured , imported
+}
+
+class Item{      // class name shoud be generic
```

Comment:
Add proper segregation for code. Don't keep everything in same file.

Main function should be in a single file and rest you can segregate into other files (make a new file for Item class).